### DISCUSSION TITLE
Support Cookie forwarding

### DISCUSSION THREAD

**Discussion by SeekDaSky** [grafana/grafana-infinity-datasource#715] [Ideas]
Grafana added proper cookie forwarding support for backend datasources since `9.4.0` according to this PR : https://github.com/grafana/grafana/pull/58132. 

I came across this [following this discussion](https://github.com/grafana/grafana-infinity-datasource/discussions/457#discussioncomment-4282515) and was wondering what the status on this was. I also thought that opening a dedicated discussion would be worth it.

Thank you very much for your hard work, this plugin is fantastic 😄

**Comment by digrich** [comment]
This support would greatly help us migrate from our current JSON data source to the Infinity Data Source. Currently, we have authentication tokens stored in cookies, and since Grafana is unable to forward these cookies to the backend service, we are unable to authenticate Infinity requests. It would be great to have this support, as all other data sources offer this.

**Comment by 18PatZ** [comment]
@yesoreyeram I also need the ability to forward certain cookies in queries. I've been able to get it working on a locally modified copy of the plugin - it's a very simple one-liner change to include the cookies and it's still protected / needs to be enabled via the Grafana `jsonData.keepCookies` settings. I'm curious whether you would be amenable to a PR to add cookie forwarding as a feature, or if you have some reasoning against it - I find it odd it's not in already due to the simplicity of the change.
